### PR TITLE
Rates rebates report

### PR DESCRIPTION
**Pull request by JacOng17**
Completed the Rates Rebates report & have compressed the images using Trimage.

### CODE REVIEW

File: _posts/2018-11-30-Rates-Rebate.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+6. [Next Steps](#Go-to-Next-Steps)
+7. [Appendix One - Service Blueprints](#Go-to-Appendix1)
+8. [Appendix Two - Trend Analysis](#Go-to-Appendix2)
+9. [Appendix Three - Comparison between the web app & current paper application](#Go-to-Appendix3)
+
+A collaborative initiative between Department of Internal Affairs and Local Authorities supported by the Service Innovation Lab 
+
+## Overview<a name = "Go-to-Overview"></a>
+
+[' A Rates Rebates’](https://www.govt.nz/browse/housing-and-property/getting-help-with-housing/getting-a-rates-rebate/#who-can-apply) is a subsidy (up to $630) for low-income homeowners on the cost of their residential rates. The subsidy is delivered by Local Councils and is administered by Central Government (specifically the Department of Internal Affairs - DIA).
```
Comment (Br3nda) [inline _posts/2018-11-30-Rates-Rebate.md]: Extra spaces and strange quotes in this title

Comment (dlouise64) [inline _posts/2018-11-30-Rates-Rebate.md]: @JacOng17  I think Brenda was referring to ```[' A Rates Rebates’]```. There is a space before A and `’` after Rebates. Should be `['A Rates Rebates']`.
Also you might want to double check it should be ['A Rates Rebate'] or ['A Rates Rebates'].

Comment (JacOng17) [reply _posts/2018-11-30-Rates-Rebate.md]: Have fixed that already :smile:  but according to the report, it's 'A Rates Rebates'
from the report

File: _posts/2018-11-30-Rates-Rebate.md [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+- __Local​ ​and​ ​Global​ ​Picture__​ - We are exploring the local and global community and identifying similar concepts, prototypes and entitlement models that we can factor into our concept development.
+
+ - __Data​ ​Path​__ - Exploring the data path of the process, data ownership, authority and declaration and what this could mean for implementation of any automated concepts.
+
+- __Thinking​ ​Big/Parallel​ ​Value​__ - How could this be applied across all of government services.
+
+## Week Two - Customer research and journey mapping
+### Service blueprint
+Service Map The Rates Rebates Process touches many agencies. We mapped the journey to see touch points for rebates customers, front stage (agency work visible to the rebate customer), and backstage (agency work not visible to the rebate customer). It showed a very complex picture with a lot of pain across users, Councils and Central government agencies. Considering this entitlement is a maximum of $610 per year, it requires a lot of cost and time from all participants in the process to administer.
+![Service Blueprint](/assets/media/Rates-Rebates/RR2.png)
```
Comment (Br3nda) [inline _posts/2018-11-30-Rates-Rebate.md]: Do you think the link below explains the image enough to people who cannot see images? Can it be cleare?

File: _posts/2018-11-30-Rates-Rebate.md [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+
+## How the alpha works - user interface design
+### Making it easy
+<img src = "/assets/media/Rates-Rebates/RR9.png" width = "425px" align = "right">To make it easy we built a web application (web app) for the ratepayer applicants to use as the basis of the alpha. Applicants can learn if they were eligible for a Rates Rebate in the current year by entering their address. We created a widget where the applicant only types their address, which is auto-completed for them, and their Rates appear - it uses publicly available data which we obtained in collaboration with the Tauranga City Council. Once the applicant enters their income and number of dependants, they get an indication of their Rates Rebate. If they provide accurate information, they will get an accurate calculation - it’s all in one place,  the user interface design guides people through the process giving them the information needed to help them make a decision whether its worthwhile for them to apply or not.
```
Comment (Br3nda) [inline _posts/2018-11-30-Rates-Rebate.md]: the image here needs a description

File: _posts/2018-11-30-Rates-Rebate.md [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+
+The web application frontend was built in ReactJS, with a jsonapi.org server created with JSONAPI Suite on Ruby on Rails. Signatures were captured using “signature_pad” npm module. The web app was built in five weeks. All components used were open source, and available on github.com for other agencies to use.
+
+### Eligibility calculator 
+Our research told us people applying for the rates rebate often don’t know how much they earn. As many are on Superannuation or Work and Income benefits we can ask if they’re on one of these income sources, as the amounts are known.
+
+We also found that there was some embarrassment about income and about asking for any form of help which was a barrier to applying, so we designed an anonymous calculation step at the beginning of the service so people don’t have to go through a full application before they find out if they are eligible for anything (which is the current process). We found users that had gone through days of effort to apply, only to discover they were entitled to 30 cents.
+
+![User journey for pilot digital application](/assets/media/Rates-Rebates/RR10.png)
```
Comment (Br3nda) [inline _posts/2018-11-30-Rates-Rebate.md]: this tells the reader it's a user journey, but we also need to provide a description, that includes the information portrayed, in the image - so users who can't see image will still be able to access the info.

**Pull request merged**